deploy/lib/Help.rb

```diff
@@ -266,6 +268,32 @@ def self.deploy
     DOC
   end
 
+  def self.merge
+    <<-DOC.strip_heredoc
+      Usage: ml {env} merge WHAT [options]
+
+      General options:
+        -v, [--verbose]  # Verbose output
+
+      Please choose a WHAT below.
+
+        content     # Merges your content db in the given environment
```

Comment:
Seems like supporting "modules" would be a trivial addition. Is it more complicated than I'm thinking? (This PR looks good, I'll accept as-is if you like.)